src/compilers/suite.cr

```diff
@@ -3,7 +3,13 @@ module Mint
     def compile(node : Ast::Suite) : Compiled
       compile node do
         location =
-          [Raw.new(node.location.to_json)]
+          [
+            Raw.new({
+              start:    {node.from[1], node.from[2]},
+              end:      {node.to[1], node.to[2]},
+              filename: node.file.path,
+            }.to_json),
```

Comment:
That looks odd to me, why don't you just pass a values into a dedicated `Raw.new` constructor instead of serializing them into JSON?

ditto elsewhere

Reply:
`Raw` means that it should render the given value in JavaScript as is, it is used in many places.